Magento_Cms/styles/module/_home.scss

```diff
@@ -0,0 +1,41 @@
+.home {
+    margin-top: 30px;
+    &__vertical-grid {
+        @include mq($screen-l) {
+            display: flex;
+            flex-direction: column;
+        }
+        & > * {
+            @include mq($screen-l) {
+                flex: 1;
+            }
+        }
+    }
+
+    &__wrapper {
+        margin-bottom: 30px;
+    }
+
+    &__column-left {
+        padding-right: 15px;
+    }
+
+    &__column-right {
+        padding-left: 15px;
+    }
+
+    &__tier-banner {
```

Comment:
sorry, I mixed up languages, I wanted to named it 'third' - like a third part banner. tier it's from french. corrected to `third-banner`